sentry/src/main/java/io/sentry/util/InitUtil.java

```diff
@@ -46,4 +52,76 @@ public static boolean shouldInit(
 
     return previousOptions.getInitPriority().ordinal() <= newOptions.getInitPriority().ordinal();
   }
+
+  public static IContinuousProfiler initializeProfiler(@NotNull SentryOptions options) {
```

Comment:
`m` should we instead check if options already have a profiler / converter set (that's not noop) and return that?
IMO that makes it easier to reason about SDK init.
Looking at the tests it seems we're potentially replacing a working profiler / converter with noop (if misusing this util).